third_party/conan/lockfiles/linux/clang7_debug/conan.lock

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
are these files generated?

Reply:
Yes

Reply:
ok, I missed the generator script in the first commit.